bugRzilla_review.Rmd

```diff
@@ -465,46 +456,73 @@ datatable(head(baa, 5), options = list(scrollX = TRUE))
 
 ### Visualizations
 
+```{r}
+#Counting number of bugs per month in an year
+bugs_counts <- baa %>%
+    arrange(bug_id) %>%
+    count(creation_year)
+skim(head(bugs_counts))
+```
+
+```{r}
+# 3D plot to see the number of bug counts per month in a year
+bug_count_graph <- plot_ly(
+    x = bugs_counts$creation_month,
+    y = bugs_counts$creation_year,
+    z = bugs_counts$n,
+    type="scatter3d", 
+    mode="markers", marker = list(size=2)) 
+
+bug_count_graph <- bug_count_graph %>%
+    layout(
+        title = "Bug Counts per month in a year"
+    )
+bug_count_graph
+```
+<font size = 4>
+    The above visualization is about the number of bugs counts per month in a year. The Most number of bug count is `77` in the `April, 2015` and the minimum bug_count is `2`.
+</font>
 ```{r}
 #filtering the data where resolution is Duplicate
 res_dupli <- baa %>%
-    filter(resolution == "DUPLICATE")
+    filter(resolution == "DUPLICATE" | bug_status == "closed")
```

Comment:
This logic operations keeps the bugs that are duplicate OR closed not duplicated AND closed as you say on text below and on the plots. You make the same mistake on multiple logic operations below. Check them and fix either the text, or better, the filter.